Sieve of Eratosthenes

```diff
@@ -0,0 +1,40 @@
+#include <bits/stdc++.h> 
+using namespace std; 
```

Comment:
Please avoid this. Include namespace prefixes instead.
Relevant: https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice